Gopkg.toml

```diff
@@ -1,65 +1,20 @@
-[[constraint]]
-  name = "github.com/docker/go"
-  version = "1.5.1-1"
-
-[[constraint]]
-  name = "github.com/stretchr/testify"
-  version = "1.2.2"
-
-[[constraint]]
-  name = "gopkg.in/yaml.v2"
-  version = "2.2.1"
-
-[[constraint]]
-  # version imported by k8s.io/client-go @ kubernetes-1.11.2
-  name = "github.com/Azure/go-autorest"
-  revision = "1ff28809256a84bb6966640ff3d0371af82ccba4"
-
-[[constraint]]
-  name = "github.com/docker/cli"
-  revision = "f95ca8e1ba6c22c9abcdbf65e8dcc39c53958bba"
-
 [[override]]
-  name = "github.com/docker/docker"
-  revision = "f76d6a078d881f410c00e8d900dcdfc2e026c841"
+  name = "github.com/containerd/containerd"
+  version = "v1.3.0"
 
 [[override]]
   name = "github.com/docker/distribution"
-  revision = "83389a148052d74ac602f5f1d62f86ff2f3c4aa5"
-
-[[override]]
-  name = "k8s.io/apimachinery"
-  revision = "kubernetes-1.11.2"
-
-[[override]]
-  name = "k8s.io/api"
-  revision = "kubernetes-1.11.2"
+  #version = "v2.7.1"
+  revision = "0d3efadf0154c2b8a4e7b6621fff9809655cc580"
 
 [[override]]
-  name = "k8s.io/client-go"
-  revision = "kubernetes-1.11.2"
-
-[[override]]
-  name = "k8s.io/kubernetes"
-  revision = "kubernetes-1.11.2"
-
-[[override]]
-  name = "k8s.io/kube-openapi"
-  revision = "d8ea2fe547a448256204cfc68dfee7b26c720acb"
-
-[[override]]
-  name = "github.com/docker/compose-on-kubernetes"
-  revision = "a6086e2369e39c2058a003a7eb42e567ecfd1f03"
-
-[[override]]
-  name = "github.com/containerd/containerd"
-  revision = "7f5f1176dd9fb3cc8d3ce5de91759ed3dc969fa2"
+  name = "github.com/docker/docker-credential-helpers"
+  version = "v0.6.3"
 
 [[override]]
   name = "golang.org/x/sys"
   revision = "f49334f85ddcf0f08d7fb6dd7363e9e6d6b777eb"
 
-[prune]
-  non-go = true
-  unused-packages = true
-  go-tests = true
+[[override]]
+  name = "github.com/google/go-containerregistry"
+  revision = "ef12d49c8daf6a6d72978966d5945e39bb898b4a"
```

Comment:
I'm wondering if we can override with a more recent commit for this repo?  Though, perhaps this was pinned at the inherited commit on master?

https://github.com/google/go-containerregistry/commit/0e086aed1d2909810e011e2a073054f2eb4e4df2 brings in a fix, courtesy @glyn, related to supporting pushing images (or indexes) with differing media types, which Porter hit with a use case mentioned in https://github.com/pivotal/image-relocation/issues/34

Reply:
I would like to defer this change to someone familiar with the expected behavior of this library (cc @glyn).